prices/providers/anthropic.yml

```diff
@@ -2,8 +2,12 @@
 name: Anthropic
 id: anthropic
 pricing_urls:
-  - https://www.anthropic.com/pricing#api
-api_pattern: 'https://api\.anthropic\.com'
+  - https://www.anthropic.com/pricing
+api_pattern: 'https://(.*\.)?anthropic\.com'
+provider_match:
+  or:
+    - equals: anthropic
+    - equals: claude
```

Comment:
is this actually a provider alias? It looks like a model name.

Reply:
true, mistake, removed